vignettes/xportr.Rmd

```diff
@@ -110,12 +109,11 @@ To create a fully compliant v5 xpt `ADSL` dataset, that was developed using R, w
 library(dplyr)
 library(labelled)
 library(xportr)
-library(admiral)
 library(rlang)
```

Comment:
could we use rename_with(tolower)

then we can drop this library(rlang).  Has always seemed a little confusing to me